config/boot.rb

```diff
@@ -1,4 +1,15 @@
 # Set up gems listed in the Gemfile.
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
 
-require 'bundler/setup' if File.exist?(ENV['BUNDLE_GEMFILE'])
+require 'bundler/setup' # Set up gems listed in the Gemfile.
+
+# load dotenv files before "before_configuration" callback
+require File.expand_path('../configuration', __FILE__)
+
+# global instance to access and use
+AppConfig = Configuration.new
+AppConfig.load_dotenv_files
+
+# set defaults to address OodAppkit.dataroot issue
+ENV['OOD_DATAROOT'] = AppConfig.dataroot.to_s
+ENV['DATABASE_PATH'] = AppConfig.database_path.to_s
```

Comment:
I can't remember if this environment variable (`DATABASE_PATH`) needs to be set. Do you happen to know what external library uses this? I checked `ood_appkit` but I can't find it there.

Reply:
removed.